plugins/catalog/src/js/repositories/RepositoriesAdd.js

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable no-unused-vars */
+import React from "react";
+/* eslint-enable no-unused-vars */
+
+import { componentFromStream } from "data-service";
+
+import { Subject } from "rxjs/Subject";
+import "rxjs/add/operator/combineLatest";
+import "rxjs/add/operator/do";
+import "rxjs/observable/empty";
+
+import { Observable } from "rxjs/Observable";
+
+import { addRepository } from "./data/repositoriesStream";
+import AddRepositoryFormModal from "./components/AddRepositoryFormModal";
+
+// Imported from the Cosmos Store
+const getErrorMessage = (response = {}) => {
+  if (typeof response === "string") {
+    return response;
+  }
+
+  return response.description || response.message || "An error has occurred.";
+};
+
+const addRepositoryEven$ = new Subject();
```

Comment:
should this say `addRepositoryEvent$`? Not sure what "even" means.

Reply:
you are right, going to fix that typo.